definitions/json/CaseEvent/CaseEvent.json

```diff
@@ -177,6 +177,21 @@
     "ShowEventNotes": "N",
     "ShowSummary": "N"
   },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "ID": "et3RespondentRepresentativeDetails",
+    "Name": "Amend contact details",
+    "Description": "Amend contact details",
+    "DisplayOrder": 53,
+    "PreConditionState(s)": "Accepted",
+    "PostConditionState": "*",
+    "CallBackURLAboutToStartEvent": "${ET_COS_URL}/et3Response/aboutToStartRepresentativeInfo",
```

Comment:
Callback is missing in et-ccd-callbacks pr - https://github.com/hmcts/et-ccd-callbacks/pull/2690